source/features/sticky-comment-header.css

```diff
@@ -0,0 +1,55 @@
+html:not([rgh-OFF-sticky-comment-header]) {
+	/* Issue body header */
+	[class^="IssueBodyHeader-module__IssueBodyHeaderContainer"],
+	/* Issue comment header */
+		[data-testid='comment-header'],
+	/* PR body/comment header */
+		.timeline-comment-header {
+		position: sticky;
+		top: calc(var(--base-sticky-header-height, 0px) + 56px);
+		z-index: 1;
+	}
+
+	/* Added by `show-names` */
+	.rgh-viewer-did-author,
+	.timeline-comment.current-user .timeline-comment-header,
+	[class*="IssueBodyHeader-module__viewerDidAuthor"] {
+		background: linear-gradient(var(--bgColor-accent-muted, fuchsia)), var(--rgh-background) !important;
+	}
+
+	/* Show menus on top of headers */
+	.js-timeline-item:has(details:open) .timeline-comment-header {
+		z-index: 2;
+	}
```

Comment:
The comment could be clearer. Without this line, it already shows on top of the header, just _not the following comments' headers._

It's useful to attach screenshots to PRs for future reference:

<img width="269" height="341" alt="Screenshot" src="https://github.com/user-attachments/assets/6d6b49bc-a323-40d8-9b56-12f39a92e843" />